src/Eccube/Resource/doctrine/migration/Version20160630110710.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160630110710 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $app = \Eccube\Application::getInstance();
+
+        $DeviceType = $app['eccube.repository.master.device_type']->find(10);
+
+        $PageLayout = $app['eccube.repository.page_layout']->findOneBy(array(
+            'url' => 'mypage_delivery_edit',
+        ));
+        if (!$PageLayout) {
+            $PageLayout = new \Eccube\Entity\PageLayout();
+            $PageLayout->setName('マイページ/お届け先編集')
```

Comment:
I'm fixed it, thanks @chihiro-adachi san for reminding. I 'll pay more attention.